### ISSUE TITLE
Who can give me some advice

### ISSUE THREAD

**Issue by CloudWise-Lukemiao** [grafana/grafana#55782]
![image](https://user-images.githubusercontent.com/76942485/192421828-3ea84c80-e677-48ec-b5aa-8ff126ff68ac.png)

Sorry, this is not a bug. I am at this URL（ https://community.grafana.com/ ）There has been no answer to the Chinese question. I want to submit a grafana plugin to the official website of grafana and make it a community version. Whether the source code and compiled dist need to be packaged into a zip when submitting. Or just dist and zip?

**Comment by tolzhabayev** [comment]
Hello @CloudWise-Lukemiao,

here is how you can submit your plugin to our catalog:
- Build and [package it](https://grafana.com/docs/grafana/latest/developers/plugins/package-a-plugin/#package-a-plugin)
- Publish it https://grafana.com/docs/grafana/latest/developers/plugins/package-a-plugin/#publish-your-plugin-on-grafanacom

TL;DR compiled dist need to be packaged properly (see above) when submitting. Once submitted we will reach out to you with further questions and feedback.

**Comment by CloudWise-Lukemiao** [comment]
> Hello @CloudWise-Lukemiao,
> 
> here is how you can submit your plugin to our catalog:
> 
> * Build and [package it](https://grafana.com/docs/grafana/latest/developers/plugins/package-a-plugin/#package-a-plugin)
> * Publish it https://grafana.com/docs/grafana/latest/developers/plugins/package-a-plugin/#publish-your-plugin-on-grafanacom
> 
> TL;DR compiled `dist` need to be packaged as a `zip` file properly (see above) when submitting. Once the url to the `zip` file is submitted we will reach out to you with further questions and feedback.

![image](https://user-images.githubusercontent.com/76942485/192482735-acd5e77b-7435-4796-8434-c0872cf60771.png)
I see that there is a code review in it, which directly submits the compiled directory. How to conduct the code review？

**Comment by tolzhabayev** [comment]
@CloudWise-Lukemiao our team (in Grafana Labs) will do a code review once you submit your plugin. So, basically, after you click submit - just wait and we will get back to you via e-mail with more questions.
